docs/userGuide/syntax/modals.mbdf

```diff
@@ -25,6 +31,11 @@ More about <trigger for="modal:trigger_id">trigger</trigger>.
   </modal>
   <br>
   This is the same <trigger for="modal:trigger_id">trigger</trigger> as last one.
+
+  <trigger for="modal:centered">This is a trigger for a centered modal</trigger>.
+  <modal title="**Centered** :rocket:" id="modal:centered" center>
```

Comment:
Looking through the `Modal` documentation, your addition looks fine. However, the previous modal example is missing the rocket emoji as well as the lorem ipsum text. The code example and output doesn't match, we have to fix that.

Since it is a minor issue, you can fix those within this PR as a separate commit.

Reply:
Also, your centered example can be improve by adding the `text-center` bootstrap class via [custom classes](https://markbind.org/userGuide/usingComponents.html#inserting-custom-classes-into-components). So everything is centered.

Reply:
Updated.  however, using slots with modals cause the ok button to go away.